inst/extdata/metadata/batch0/platemap/C-7161-01-LM6-001.txt

```diff
@@ -0,0 +1,385 @@
+plate_map_name	well_position	broad_sample	mg_per_ml	mmoles_per_liter	solvent	Metadata_broad_sample_type	broad_sample_type
```

Comment:
`Metadata_broad_sample_type`	and `broad_sample_type` are not present in the platemaps we receive from CDOT

Reply:
Further, because of the duplicate columns, it gets propagated to other files 
e.g. `SQ00015116_augmented.csv` has 
`Metadata_Metadata_broad_sample_type`,`Metadata_broad_sample_type`